it/stubs/PenaltiesStub.scala

```diff
@@ -182,6 +184,8 @@ object PenaltiesStub {
         penaltyStatus = LPPPenaltyStatusEnum.Accruing,
         penaltyAmountPaid = Some(1001.45),
         penaltyAmountOutstanding = Some(99.99),
+        penaltyAmountPosted = 1001.45,
```

Comment:
This is an accruing penalty, therefore there would be no posted amount. Outstanding amount would be None and paid would also be None